tests/Unit/Factories/ResultsRendererFactoryTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rokas
+ * Date: 17.10.20
+ * Time: 23.06
+ */
+
+namespace Churn\Tests\Results;
+
+use Churn\Factories\ResultsRendererFactory;
+use Churn\Renderers\Results\ConsoleResultsRenderer;
+use Churn\Renderers\Results\JsonResultsRenderer;
+use Churn\Results\ResultCollection;
+use Churn\Tests\BaseTestCase;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ResultsRendererFactoryTest extends BaseTestCase
+{
+    /**
+     * @var JsonResultsRenderer|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $jsonResultsRenderer;
+
+    /**
+     * @var ConsoleResultsRenderer|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $consoleResultsRenderer;
+
+    /**
+     * @var ResultsRendererFactory
+     */
+    private $resultsRendererFactory;
+
+    public function setUp()
+    {
+        $this->jsonResultsRenderer = $this->createMock(JsonResultsRenderer::class);
+        $this->consoleResultsRenderer = $this->createMock(ConsoleResultsRenderer::class);
+
+        $this->resultsRendererFactory = new ResultsRendererFactory(
+            $this->jsonResultsRenderer,
+            $this->consoleResultsRenderer
+        );
+    }
+
+    /**
+     * @test
+     */
+    public function it_can_be_created()
+    {
+        $this->assertInstanceOf(ResultsRendererFactory::class, $this->resultsRendererFactory);
+    }
+
+    /**
+     * @test
+     * @expectedException \InvalidArgumentException
```

Comment:
Instead of relying on `@expectedException` could you please use the PHPUnit method that checks for the exception instead? I believe it is `$this->expectException()`